nservicebus/recoverability/configure-delayed-retries.md

```diff
@@ -28,24 +29,21 @@ snippet:DisableDelayedRetries
 
 ## Custom Retry Policy
 
-Custom retry logic can be configured based on headers or timing in code.
-
-
-### Applying a custom policy
+Custom retry logic can be configured via code.
 
 snippet:DelayedRetriesCustomPolicy
 
 
 ### Simple Policy
 
-The following retry policy that will retry a message 3 times with a 5 second interval.
+The following retry policy overrides default delay interval and sets it to 5 seconds.
 
 snippet:DelayedRetriesCustomPolicyHandler
 
 
 ### Exception based Policy
 
-The following retry policy extends the previous policy with a custom handling logic for a specific exception.
+The following retry policy extends the previous one by skipping delayed retries whenever `MyBusinessException` has been thrown during incoming message processing.
```

Comment:
Can you add a brief intro to the section header. For example: 

Sometimes the number of retries or the delay interval might depend on the error exception thrown. In which case, it might be useful to define a policy that's based on the exception received. In the following example, whenever `MyBusinessException` is thrown when a message is processed, the message will be delayed for ... [explain how long is the delay without referring to the previous example.]